web/src/settings_components.ts

```diff
@@ -1576,26 +1576,29 @@ export function create_group_setting_widget({
     return pill_widget;
 }
 
-type realm_group_setting_name =
-    | "can_add_custom_emoji_group"
-    | "can_create_groups"
-    | "can_create_public_channel_group"
-    | "can_create_private_channel_group"
-    | "can_delete_any_message_group"
-    | "can_delete_own_message_group"
-    | "can_manage_all_groups"
-    | "can_move_messages_between_channels_group"
-    | "can_move_messages_between_topics_group"
-    | "create_multiuse_invite_group"
-    | "direct_message_initiator_group"
-    | "direct_message_permission_group";
+export const realm_group_setting_name_schema = z.enum([
```

Comment:
Do we want to name this explaining that this not includes setting which allow only system groups?

Reply:
I think we can discuss our path forward for these; might not be worth spending a lot of attention on naming if we expect to remove the component.